Projects/hCore/Sources/UIImage+imageView.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  UIImage+imageView.swift
+//  hCore
+//
+//  Created by Sam Pettersson on 2021-04-23.
+//  Copyright © 2021 Hedvig AB. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+extension UIImage {
+    public func imageView(height: CGFloat? = nil, width: CGFloat? = nil) -> UIImageView {
```

Comment:
Shouldn't this be in `hCoreUI` I think UIKit extensions we should try to keep in `hCoreUI` and `foundation` extensions or custom helpers in `hCore`?